src/scripts/worldmap/MapHelper.ts

```diff
@@ -16,6 +16,30 @@ enum areaStatus {
 
 class MapHelper {
 
+    public static generateMapPopoverContent(): string {
+        const rows = [];
+        GameHelper.enumStrings(areaStatus).map(status => Settings.getSetting(`--${status}`))
+            .filter(setting => setting.isUnlocked()).forEach(setting => {
+                rows.push(
+                    `<div class='col-10 legend-item'>
+                        <span>${setting.displayName}:</span>
+                    </div>
```

Comment:
I don't really like HTML code in the typescript files. This generated code can easily be done with knockout in the html file